### DISCUSSION TITLE
hvplot to html

### DISCUSSION THREAD

**Discussion by ellenalamont17** [earthlab-education/Earth-Analytics-2023-01-Intro#476] [Q&A04 Raster Data]
Has anyone had success getting your hvplots in the Raster Data exercise to export properly in html? I know it can be done with matplotlib, but I am specifically looking for an hvplot solution. Currently, my plots work perfectly in the codespace. When I run the html export, my map appears but my plots are missing. 

Searching through the stack exchange and similar sites, I've tried a few solutions:
- Setting _dynamic=False_ to turn off the interactive features of the plots
- Exporting the plot as an independent html and pulling it into the portfolio page as a weblink
- importing holoview and trying some of that export functionality.

I've not had success with any of these.  Anyone have thoughts on this?

**Comment by eculler** [comment]
Hi @ellenalamont17 - when you say that your plots are missing, can you clarify what you mean? Namely, the hvplot plots will never show up in the preview in your GitHub repository. However, mine have so far been showing up on the actual web page, if they are showing up in Codespaces.

**Reply by ellenalamont17** [reply]
@eculler - To clarify, when I created my code in codespace, the plots appear in the codespace with no issues. When I convert my codespace to an html and obtain the proper URL path (e.g. https://ellenalamont17.github.io/notebook/ndvi_fire_CRG.html) all of my interactive plots are missing. The images and weblinks appear, but the plots do not. I've tried several modifications as described above, but it does not seem to change.

**Reply by eculler** [reply]
@ellenalamont17 ok, hmm. I'm not quite sure why that happens. Can you link me to your repo so I can muck around a bit? I think other folks are having this problem, but it seems intermittent.

**Reply by ellenalamont17** [reply]
@eculler You should have an invite to my repo.
